project/src/pages/favorites/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {FC} from 'react';
+import {OffersList} from '../../components/OffersList/OffersList';
+import {CardProps} from '../../components/App/App';
+import {FavoritesList} from '../../components/FavoritesList/FavoritesList';
+
+type FavoritesProps = {
+  offers: CardProps[]
+}
+
+export const Favorites:FC<FavoritesProps> = ({offers}) => (
```

Comment:
```suggestion
export const Favorites = ({offers}: FavoritesProps) => (
```

а зачем предыдущий файл?